requirements/base.txt

```diff
@@ -257,7 +257,7 @@ sqlalchemy==1.3.24
     #   flask-sqlalchemy
     #   marshmallow-sqlalchemy
     #   sqlalchemy-utils
-sqlalchemy-utils==0.36.8
+sqlalchemy-utils==0.37.8
```

Comment:
`sqlalchemy-utils==0.36.8` has been yanked from PyPI, causing an ugly warning when installing deps. In addition, `py39` was added to the officially supported versions right after `0.37.8` was cut (https://github.com/kvesteri/sqlalchemy-utils/commit/b5b48b13a22aa6d796e98c0225f367a2f010c788), so I assume `0.37.8` to be fully supported by 3.9 in practice.